src/diagnostics/qt/diWidget.hh

```diff
@@ -23,7 +21,15 @@ class diWidget : public QWidget {
 
 public:
   explicit diWidget(QWidget *parent = 0);
-  ~diWidget(){/* empty */};
+  ~diWidget(){};
```

Comment:
Are we not doing the “empty” or “...” for empty braces any more? I thought I remembered this being the suggested style, but I could be wrong.

Reply:
* I like `/* empty */`, but many static analyzers flag this.  The modern preferred version is `~diWiedget() = default;`.  Let me try that and push an update.

Reply:
Fixed in 8df05e3.